src/TypeScript/Analyzer/TypeScriptAnalyzer.cs

```diff
@@ -80,10 +72,10 @@ public bool IsAnalysisSupported(IEnumerable<AnalysisLanguage> languages)
         {
             Debug.Assert(IsAnalysisSupported(detectedLanguages));
 
-            ExecuteAnalysis(path, consumer, cancellationToken).Forget(); // fire and forget
+            ExecuteAnalysisAsync(path, consumer, cancellationToken).Forget(); // fire and forget
         }
 
-        internal async Task ExecuteAnalysis(string filePath, IIssueConsumer consumer, CancellationToken cancellationToken)
+        internal async Task ExecuteAnalysisAsync(string filePath, IIssueConsumer consumer, CancellationToken cancellationToken)
```

Comment:
ts config here is only used in the NullOrEmpty check and is not passed to the ExecuteAnalysis method (see (1))